### ISSUE TITLE
if only epoch 200 or 500 times, should I change the 'num_images' as well?

### ISSUE THREAD

**Issue by sumorday** [w86763777/pytorch-gan-collections#15]
Hi! Dear Yi-Lun
I want to epoch 200 times or 500 times because 10k or 50k will cost a long time to run.
I changed the sample_step=50 and total_steps=200
should I change num_images=50000 as num_images = 50? I guess it should be smaller than epoch times? or it doens't matter ?
Thank you in advance:)

--arch=cnn32
--batch_size=128
--dataset=cifar10
--fid_cache=./stats/cifar10.train.npz
--logdir=./logs/SNGAN_CIFAR10_CNN
--loss=hinge
--lr_D=0.0002
--lr_G=0.0002
--n_dis=1
**--num_images=50**
--record
**--sample_step=50**
--sample_size=64
--seed=0
**--total_steps=500**
--z_dim=100

**Comment by w86763777** [comment]
- `total_steps`: The number of generator updates of this training.
- `num_images`: The number of images which is used to calculate Inception Score and FID.
- `sample_step`: Sample some images and save them to disk and tensorboard every this steps.

These three parameters are not correlated.

**Comment by sumorday** [comment]
@w86763777 
謝謝您！明白了！
Thank you soooo much!
